Description: Port sm_capture.sh to Debian
 Use dpkg-query to determine package version. Use base instead of sh.
Author: Brian T. Smith <bsmith@systemfabricworks.com>=
Forwarded: no
Last-Update: <2018-06-23>
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Esm/ib/src/linux/startup/sm_capture.sh
+++ b/Esm/ib/src/linux/startup/sm_capture.sh
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 # BEGIN_ICS_COPYRIGHT8 ****************************************
 # 
 # Copyright (c) 2015-2017, Intel Corporation
@@ -119,7 +119,11 @@
 fi
 
 echo "Getting FM rpm version..."
+if [ -f /etc/debian_version ] ; then
+	dpkg-query -W --showformat '${db:Status-Abbrev} ${Package} {$Version}' | egrep 'opa-fm' > $dumpLocation/version
+else
 rpm -qa|egrep 'opa-fm' > $dumpLocation/version
+fi
 
 echo "Copying FM configuration..."
 cp $CONFIG_FILE $dumpLocation
